docs/search/copilot.md

```diff
@@ -0,0 +1,155 @@
+---
+id: copilot
+title: Sumo Logic Copilot - Feature Preview
+sidebar_label: Copilot 🤖
+keywords:
+  - copilot
+  - artificial intelligence
+  - ai
+  - machine learning
+  - ml
+---
+
+<head>
+  <meta name="robots" content="noindex" />
+</head>
+
+import Iframe from 'react-iframe';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+:::info
+This is a Preview release. To learn more, contact your Sumo Logic account executive.
+:::
+
+Sumo Logic Copilot is an AI-based assistant that streamlines log analysis insights by allowing you to ask questions in plain English and provides search suggestions, all without your having to write a log query.
+
+In this document, you'll learn the recommended Copilot workflow as well as best practices.
+
+<details>
+<summary>Click here to watch a step-by-step video investigation using Copilot.</summary>
+
+<!-- add micro lesson video when published-->
+
+<Iframe url="https://player.vimeo.com/video/939372059?badge=0&amp;autopause=0&amp;player_id=0&amp;app_id=58479"
+        width="854px"
+        height="480px"
+        title="Copilot Demo"
+        id="myId"
+        className="video-container"
+        display="initial"
+        position="relative"
+        allow="accelerometer; autoplay=1; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+        allowfullscreen
+        />
+
+Scenario: We receive a Sumo Logic alert about AWS CloudTrail access keys being leaked outside our organization and use Copilot to investigate.
```

Comment:
Add note above this stating the steps below are meant to replace the audio. (Video has no audio)